ImperatorToCk3Tests/ImperatorWorldTests/Genes/GenesDBTests.cpp

```diff
@@ -0,0 +1,52 @@
+#include "gtest/gtest.h"
+#include <sstream>
+
+
+#include "../ImperatorToCK3/Source/Imperator/Genes/GenesDB.h"
+#include "../ImperatorToCK3/Source/Imperator/Genes/AccessoryGenes.h"
+
+
+TEST(ImperatorWorld_GenesTests, genesDefaultToEmpty)
+{
+	std::stringstream input;
+	input << "=\n";
+	input << "{\n";
+	input << "}";
+
+	const ImperatorWorld::GenesDB genes(input);
+
+	//ASSERT_TRUE(genes.getMorphGenes().empty());
+	ASSERT_TRUE(genes.getAccessoryGenes().getGenes().empty());
+}
+
+TEST(ImperatorWorld_GenesTests, accessoryGenesCanBeLoadedInsideGeneGroup)
+{
+	std::stringstream input;
+	input << "accessory_genes = {\n";
```

Comment:
It's not too bad here, but check what I've done here to make an easier-to-build construct for tests: https://github.com/ParadoxGameConverters/Vic2ToHoI4/blob/master/Vic2ToHoI4/Source/V2World/Pops/PopBuilder.h

Reply:
Will try it on Pops first.